Sources/Commands/SwiftPackageTool.swift

```diff
@@ -40,6 +61,8 @@ public enum PackageMode: Argument, Equatable, CustomStringConvertible {
             self = .fetch
         case "generate-xcodeproj":
             self = .generateXcodeproj
+        case "get-package-path":
+            self = .getPackagePath
```

Comment:
I meant by just loading the workspace and reading out the package path, you could use the same code which implements `get-package-path`, just not expose it yet.